### PR TITLE
feat: [ch5639] published at range in api responses

### PR DESCRIPTION
**Pull request by marcinaylien**
I am not sure if I miss smth :)

### CODE REVIEW

File: samples/java/target [hunk 1]
```diff
@@ -1 +0,0 @@
-../../sdks/news-api/java/target
```
Comment (mdibaiee) [inline samples/java/target:1]: Can you please bring this back? This is important for samples to work

Comment (mjakubowski84) [reply samples/java/target:1]: dammit, it was deleted by IDE automatically as was broken, didn't notice that

Comment (marcinaylien) [reply samples/java/target]: it is back

**Comment by fcibecchini** [comment]
>The end published date of the time series

That should be histogram/trend/etc. I suppose

**Comment by marcinaylien** [comment]
> > The end published date of the time series
> 
> That should be histogram/trend/etc. I suppose

what do you mean?

**Comment by javiramos1** [comment]
> > > The end published date of the time series
> > 
> > 
> > That should be histogram/trend/etc. I suppose
> 
> what do you mean?



> > > The end published date of the time series
> > 
> > 
> > That should be histogram/trend/etc. I suppose
> 
> what do you mean?

Fabio, Marcin added them to histograms, trends...

**Comment by fcibecchini** [comment]
Sorry, I mean in the description field

**Comment by marcinaylien** [comment]
You refer to some other existing field (though similar). TBH I have problem with understanding this description.
What description do you suggest for all of them?

**Comment by mdibaiee** [comment]
@marcinaylien just to make sure this is valid, can you try running the SDK generation script? If it runs without errors we are okay

**Comment by marcinaylien** [comment]
@fcibecchini I fixed my copy-paste error
@mdibaiee sdk generates (and post-generates) without error

**Pull request merged**